c/src/ml-api-service-offloading.h

```diff
@@ -32,6 +32,36 @@ extern "C" {
  */
 int ml_service_offloading_create (ml_service_h handle, ml_option_h option);
 
+/**
+ * @brief Start ml offloading service.
+ * @remarks The @a handle should be destroyed using ml_service_destroy().
+ * @param[in] handle ml-service handle created by ml_service_new().
+ * @return @c 0 on Success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #ML_ERROR_STREAMS_PIPE Failed to launch the pipeline.
+ * @retval #ML_ERROR_TRY_AGAIN The pipeline is not ready yet.
+ * @retval #ML_ERROR_PERMISSION_DENIED The application does not have the privilege to access to the storage.
+ */
+int ml_service_offloading_start (ml_service_h handle);
```

Comment:
Add ml_service_offloading_start, stop to L128 when ENABLE_SERVICE_OFFLOADING is disabled.
```
#define ml_service_offloading_start(...) ML_ERROR_NOT_SUPPORTED
```

Reply:
Thank you.